core/src/style/pointStyleBuilder.cpp

```diff
@@ -138,11 +138,17 @@ auto PointStyleBuilder::applyRule(const DrawRule& _rule, const Properties& _prop
     _rule.get(StyleParamKey::angle, p.labelOptions.angle);
 
     auto sizeParam = _rule.findParameter(StyleParamKey::size);
-    if (sizeParam.stops && sizeParam.value.is<float>()) {
-        float lowerSize = sizeParam.value.get<float>();
-        float higherSize = sizeParam.stops->evalWidth(m_zoom + 1);
```

Comment:
Don't know why `evalWidth` was used before here? I have switched it to use `evalFloat` both in `pointStyleBuilder` and `Stops::evalSize`.
Or I am missing something?